x/programs/v2/import_log.go

```diff
@@ -0,0 +1,28 @@
+package v2
+
+import (
+	"fmt"
+	"os"
+)
+
+func NewLogModule() *ImportModule {
+	return &ImportModule{name: "log",
```

Comment:
🤔 I think this is overkill, but I could be convinced otherwise. What we have right now is something that will log strings. It's super useful, because you want use the `wasmlanche_sdk::dbg!` macro that works just like the `std::dbg!` macro. It will inject the file-path and line-number as a prefix to whatever you're logging. It's not mean to be used with any environment variable log-level type thing.

Might need to discuss this a little more regardless.

On another node, is it possible to conditionally compile in Go? There should be no logging on a production VM.

Reply:
`// +build tag`
makes the code only compiles when 'tag' is attached to the build

Reply:
I've moved this to single entry, but I'm not going to worry about noop'ing this on the host side.  Just calling into the host is expensive, so I'll leave it is up to the guest code to noop in release.